cli/cmd.js

```diff
@@ -179,6 +182,69 @@ commander
 
   });
 
+commander
+  .command('upgradekvm')
+  .option('-o, --org <org>', 'the organization')
+  .option('-e, --env <env>', 'the environment')
+  .option('-u, --username <user>', 'username of the organization admin')
+  .option('-p, --password <password>', 'password of the organization admin')
+  .option('-v, --virtualhost <virtualhost>', 'virtual host of the proxy')
+  .option('-b, --baseuri <baseuri>', 'baseuri for management apis')  
+  .description('upgrade kvm to support JWT Key rotation')
+  .action((options)=>{
+    options.error = optionError;
+    if (!options.username) { return options.error('username is required'); }
+    if (!options.org) { return options.error('org is required'); }
+    if (!options.env) { return options.error('env is required'); }
+    promptForPassword(options,(options)=>{
+      if (!options.password) { return options.error('password is required'); }
+      upgradekvm.upgradekvm(options, () => {
+      });      
+    })
+  });
+
+  commander
+    .command('upgradeauth')
+    .option('-o, --org <org>', 'the organization')
+    .option('-e, --env <env>', 'the environment')
+    .option('-u, --username <user>', 'username of the organization admin')
+    .option('-p, --password <password>', 'password of the organization admin')
+    .option('-v, --virtualhost <virtualhost>', 'virtual host of the proxy')
+    .option('-b, --baseuri <baseuri>', 'baseuri for management apis')  
+    .description('upgrade edgemicro-auth proxy')
+    .action((options)=>{
+      options.error = optionError;
+      if (!options.username) { return options.error('username is required'); }
+      if (!options.org) { return options.error('org is required'); }
+      if (!options.env) { return options.error('env is required'); }
+      promptForPassword(options,(options)=>{
+        if (!options.password) { return options.error('password is required'); }
+        upgradeauth.upgradeauth(options, () => {
+        });      
+      })
+    });
+
+    commander
+      .command('rotatekey')
+      .option('-o, --org <org>', 'the organization')
+      .option('-e, --env <env>', 'the environment')
+      .option('-u, --username <user>', 'username of the organization admin')
+      .option('-p, --password <password>', 'password of the organization admin')
+      .option('-k, --kid <kid>', 'new key identifier')
+      .option('-b, --baseuri <baseuri>', 'baseuri for management apis')
```

Comment:
we should add a -r for the runtime url to access the edge auth proxy. https://domain:9001